gammapy/cube/psf_kernel.py

```diff
@@ -264,3 +266,44 @@ def from_gauss(
     def write(self, *args, **kwargs):
         """Write the Map object which contains the PSF kernel to file."""
         self.psf_kernel_map.write(*args, **kwargs)
+
+    def make_image(self, exposure, spectrum=None):
+        """ Make a 2D PSF from a PSF kernel
+
+        The PSF Kernel is first weighed with a spectrum and an exposure.
+        The PSF is also normalised after summation.
+
+        Parameters
+        ----------
+
+        exposure : Pre-computed exposure map
```

Comment:
Technically one should not use an exposure map to average the PSF but simply a vector of exposures at the typical tested position. So here you could pass either an array of exposures for the same true energies.